src/enums/filingCodes.ts

```diff
@@ -12,5 +12,7 @@ export enum FilingCodes {
   DISSOLUTION_COURT_ORDERED_LIQUIDATION = 'DIS_COLQD',
   DISSOLUTION_RESTORED_IN_ERROR = 'DIS_RSTR',
   SPECIAL_RESOLUTION = 'SPRLN',
-  AFFIDAVIT = 'AFDVT'
+  AFFIDAVIT = 'AFDVT',
+  REGISTRATION_SP = 'FRREG',
+  REGISTRATION_GP = 'FRREG'
```

Comment:
Does these really have the same code?

Reply:
Indeed
See ticket: https://app.zenhub.com/workspaces/entities-team-space-6143567664fb320019b81f39/issues/bcgov/entity/9251

Reply:
In the comments, Linda posts about her research into these.

Reply:
ok